apps/desktop-wallet/src/storage/addresses/addressesStorageUtils.ts

```diff
@@ -50,7 +49,6 @@ export const saveNewAddresses = (addresses: AddressBase[]) => {
 
   store.dispatch(newAddressesSaved(addresses))
   store.dispatch(syncAddressesData(addressHashes))
-  store.dispatch(syncAddressesAlphHistoricBalances(addressHashes))
```

Comment:
I'm so happy that we can simply remove this line and the component will take care of making a new request when it needs to, imperatively :D

Reply:
Great feeling eh!